Package.swift

```diff
@@ -41,28 +41,24 @@ let package = Package(
             name: "Ice",
             dependencies: ["IceImpl"],
             path: "swift/src/Ice",
-            exclude: ["slice-plugin.json"],
```

Comment:
The plug-in adds the config file to the build commands input. This avoid the need to exclude, and ensures compilation run when the configuration changes.